test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -90,23 +91,65 @@ const (
 	configFile               = "config/performance-config.yaml"
 	extensionPointsLabelName = "extension_point"
 	resultLabelName          = "result"
+	pluginLabelName          = "plugin"
 )
 
 var (
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
-		Metrics: map[string]*labelValues{
+		Metrics: map[string][]*labelValues{
```

Comment:
I wonder if the metrics in scheduler-perf are under control like other metrics. It means we should list stable metrics.
https://kubernetes.io/docs/reference/instrumentation/metrics/#list-of-stable-kubernetes-metrics

WDYT? @logicalhan

Reply:
This is a bit out of scope in this PR but let me ask you a question for our future.

Reply:
I don't think we're going to be able to parse this file..

Reply:
It looks... challenging from a static analysis perspective.

Reply:
Probably I don't understand what @utam0k means. Why should we in the first place actually? 
All metrics used in scheduler_perf are existing metrics that are exposed from scheduler, not something like scheduler_perf's original metrics.

Reply:
Sorry you are right.